pkg/util/mount/mount.go

```diff
@@ -152,6 +151,32 @@ func GetMountRefs(mounter Interface, mountPath string) ([]string, error) {
 	return refs, nil
 }
 
+// GetMountRefsByDev finds all references to the device provided
+// by devPath; returns a list of paths.
+func GetMountRefsByDev(mounter Interface, globalPath string) ([]string, error) {
```

Comment:
since this is a globally visible utility, need a unit test to make sure it works as designed.